tests/application/containers/syslog/syslog.Dockerfile

```
@@ -2,9 +2,22 @@ FROM alpine:latest
 
 SHELL ["/bin/sh", "-euo", "pipefail", "-c"]
 
-RUN mkdir -p /var/log/remote && \
-    apk --no-cache add rsyslog net-tools
+RUN \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)